test/tools/d_do_test.d

```diff
@@ -674,7 +674,7 @@ int tryMain(string[] args)
     Result testCombination(bool autoCompileImports, string argSet, size_t permuteIndex, string permutedArgs)
     {
         string test_app_dmd = test_app_dmd_base ~ to!string(permuteIndex) ~ envData.exe;
-
+        string lastCommand;
```

Comment:
I suggest adding a simple comment here describing what this is for, so the next person to modify this code doesn't have to do so much investigation.